src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -24,12 +24,17 @@ public static class ScriptConstants
         public const string TracePropertyEventDetailsKey = "MS_EventDetails";
         public const string TracePropertyIsUserTraceKey = "MS_IsUserTrace";
         public const string TracePropertyIsSystemTraceKey = "MS_IsSystemTrace";
+        public const string TracePropertyFunctionInvocationIdKey = "MS_FunctionInvocationId";
+        public const string TracePropertyScriptHostInstanceIdKey = "MS_ScriptHostInstanceId";
+        public const string TracePropertyActivityIdKey = "MS_ActivityId";
 
         public const string TraceSourceSecretManagement = "SecretManagement";
         public const string TraceSourceHostAdmin = "HostAdmin";
         public const string TraceSourceFileWatcher = "FileWatcher";
         public const string TraceSourceSwagger = "SwaggerDocumentation";
         public const string TraceSourceHttpHandler = "HttpRequestTraceHandler";
+        public const string TraceSourceScriptHost = "Script.Host";
```

Comment:
These values are going to end up in peoples logs, and I'm not sure what value the prefix "Script." adds. Notice that none of the other source values above have a prefix on them.

Reply:
We do this prefix in webjobs SDK: [TraceSource](https://github.com/Azure/azure-webjobs-sdk/blob/663a508e8a851629c26a51e7de3af36629dfd120/src/Microsoft.Azure.WebJobs.Host/TraceSource.cs#L24).